src/styles/droneList.css

```diff
@@ -0,0 +1,25 @@
+ol {
```

Comment:
I would also play a little bit with the css and maybe make the list of drones to look more like the design on figma

Reply:
Will start from tomorrow